<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditCustomer</class>
 <widget class="QDialog" name="EditCustomer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>611</width>
    <height>210</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>611</width>
    <height>210</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>611</width>
    <height>210</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="nameLabel">
           <property name="text">
            <string>Customer name:</string>
           </property>
           <property name="buddy">
            <cstring>nameLineEdit</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="addressLabel">
           <property name="text">
            <string>New Address:</string>
           </property>
           <property name="buddy">
            <cstring>nameLineEdit</cstring>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLineEdit" name="nameLineEdit"/>
         </item>
         <item>
          <widget class="QLineEdit" name="addressLineEdit"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <property name="spacing">
        <number>20</number>
       </property>
       <property name="leftMargin">
        <number>20</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Importance Level:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="doNotChange_2">
         <property name="text">
          <string>Do Not Change</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">importanceLevelButtonGroup</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="keyToHaveRadioButton_2">
         <property name="text">
          <string>Key</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">importanceLevelButtonGroup</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="niceToHaveRadioButton_2">
         <property name="text">
          <string>Nice to Have</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">importanceLevelButtonGroup</string>
         </attribute>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <property name="spacing">
        <number>20</number>
       </property>
       <property name="leftMargin">
        <number>20</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Interest Level:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="doNotChange">
         <property name="text">
          <string>Do Not Change</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="veryInterestedRadioButton_2">
         <property name="text">
          <string>Very Interested</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="somewhatInterestedRadioButton_2">
         <property name="text">
          <string>Somewhat Interested</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="notInterestedRadioButton_2">
         <property name="text">
          <string>Not Interested </string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="acceptPushButton">
     <property name="text">
      <string>Accept</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="cancelPushButton">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="importanceLevelButtonGroup"/>
 </buttongroups>
</ui>
